File: dotnet/Allmark/Utils/SkipSpaces.cs
namespace Allmark;

public static partial class Utils
{
	public static int SkipSpaces(string text, int start)
	{
		for (; start < text.Length; start++)
		{
			if (!IsSpace((int)text[start]))
			{
				break;
			}
		}
		return start;
	}
}

File: swift/Sources/Allmark/Utils/SkipSpaces.swift
import Foundation

func skipSpaces(text: [Character], start: Int) -> Int {
	var index = start
	while index < text.count {
		if !isSpace(code: Int(text[index].asciiValue ?? 0)) {
			break
		}
		index += 1
	}
	return index
}
